README.rst

```diff
@@ -125,9 +126,12 @@ a JSON file that contains two attributes:
 
 * ``codebooks``: a map from input file name to codebook file name, e.g., ``{"individual.tsv": "codebook.txt"}``.
 
-See `<test_data/input_data/config/sources_config.json>`_ for an example.
+* ``file_format``: a map from input file name to file format options, e.g., ``{"individual.tsv": {"delimiter": ","}}``.
```

Comment:
Maybe also useful to mention that if not specified, it will use `\t` as default?